grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -743,16 +743,26 @@ def section_streaming(outer_panels):
                     ],
                 ),
                 panels.timeseries_count(
-                    "Kafka Consumer Lag Size",
-                    "Kafka Consumer Lag Size by source_id, partition and actor_id",
+                    "Kafka high watermark and source latest message",
+                    "Kafka high watermark by source and partition and source latest message by partition, source and actor",
                     [
                         panels.target(
                             f"{metric('source_kafka_high_watermark')}",
-                            "source={{source_id}} partition={{partition}}",
+                            "high watermark: source={{source_id}} partition={{partition}}",
                         ),
                         panels.target(
                             f"{metric('source_latest_message_id')}",
-                            "source={{source_id}} partition={{partition}} actor_id={{actor_id}}",
+                            "latest msg: source={{source_id}} partition={{partition}} actor_id={{actor_id}}",
+                        ),
+                    ],
+                ),
+                panels.timeseries_count(
+                    "Kafka Consumer Lag Size",
+                    "Kafka Consumer Lag Size by source_id, partition and actor_id",
+                    [
+                        panels.target(
+                            f"{metric('source_kafka_high_watermark')} - on(source_id, partition) group_right() {metric('source_latest_message_id')}",
```

Comment:
Due to misalignment and non-atomicity of metrics reporting, it is possible to have source_latest_message_id > source_kafka_high_watermark. Therefore, I recommend cap the result to 0:
```
f"{clamp_min(metric('source_kafka_high_watermark')} - on(source_id, partition) group_right() {metric('source_latest_message_id')}, 0)"
```